ansibullbot/triagers/ansible.py

```diff
@@ -1545,7 +1545,7 @@ def move_issue(self, issue):
         # cache the data in case of errors
         mdata = self.IM.migration_map.get(issue.html_url)
         with open(mfile, 'wb') as f:
-            f.write(json.dumps(mdata, indent=2))
+            f.write(json_dumps(mdata))
```

Comment:
```suggestion
            f.write(to_bytes(json_dumps(mdata), errors='surrogate_or_strict'))
```

The file is opened for writing in binary mode.  So you always want to write bytes to the file, not text.

Could you do this the other way and open the file in non-binary mode?  That won't work on python2 as open in python2 always expects bytes.  So you can get UnicodeError tracebacks if you write text to the filehandle on python2 when python2 implicitly tries to convert from text to bytes.